testing-standard.md

```diff
@@ -1,20 +1,21 @@
 # HX Testing Standards
 
 # 1.0 Unit Tests
+The aim is for our unit test output to describe the functionality of our code. As long as our tests are readable and clearly structured, we should focus our time on ensuring we have high quality tests. Being able to use the test output as a description for a piece of work, where appropriate, ensures our code is unambiguous and that are tests cover expected functionality. We _should_ be able to use our test output as documentation to our code.
 
 ## 1.1 Our Technology
 In order to keep test writing familiar across the Holiday Extras group, we should use the following tech for writing our unit tests:
 
 * [Mocha](https://mochajs.org/): Our test framework of choice
 * [Sinon](http://sinonjs.org/): For our stubs & spies
-* [Chai](http://chaijs.com/): For making assertions in our tests
-  * [Sinon-chai](https://github.com/domenic/sinon-chai): For integrating Sinon with Chai
-  * [chai-as-promised](https://github.com/domenic/chai-as-promised/): For clean promise assertions
+* [Assert](https://www.npmjs.com/package/assert) or [Chai](http://chaijs.com/): For making assertions in our tests
```

Comment:
You've added `Assert` back in here, but the rest of the documentation does not reference it. I think that if it is to be recommended here you should also talk about it and provide examples further on.
I'd also be interested to hear the case for adding it back in, what was the issue without it and what does it solve by adding back in?